nixos/modules/services/misc/apache-kafka.nix

```diff
@@ -95,10 +95,7 @@ in {
         "-Xmx1G"
         "-Xms1G"
         "-XX:+UseCompressedOops"
-        "-XX:+UseParNewGC"
-        "-XX:+UseConcMarkSweepGC"
-        "-XX:+CMSClassUnloadingEnabled"
-        "-XX:+CMSScavengeBeforeRemark"
+        "-XX:+UseG1GC"
```

Comment:
I wonder if there's a need to be explicit since G1 should be default GC since java 9.

Reply:
https://github.com/apache/kafka/blob/aba9036eb681597911343510e246de4fc547a479/bin/kafka-run-class.sh#L257. I think, the option of default should be matched with this line of the shell.